neostatus/__main__.py

```diff
@@ -17,6 +20,11 @@ def main():
             status.drive()
         case 'safemode':
             status.safemode()
+        case 'set':
+                if args.red is None or args.green is None or args.blue is None:
+                    print("-r, -g and -b must be specified when setting a color")
+                    exit()
```

Comment:
Does this exit work? I would assume `import sys` and `sys.exit(1)` was required?

Reply:
Yes, it's a synonym for quit()
See https://stackoverflow.com/a/71114073 for more info
Example outout:
```
[acr@acr-desktop ~/git/neostatus]$ python -m neostatus set -r 255 -g 0
-r, -g and -b must be specified when setting a color
[acr@acr-desktop ~/git/neostatus]$

Reply:
Learned something new. Please add the non-zero exit code, though, so callers know there was a problem.